web/src/styles/sections/login-email-check.scss

```diff
@@ -0,0 +1,18 @@
+.fit-screen-content {
+  background-color: $cool-extra-light;
+}
```

Comment:
Styling applied to the parent `<div>` due to this gap between the 2 containers I observed when attempting to apply this rule to the email check `<main>`

<img width="739" alt="Screenshot 2024-12-13 at 11 48 49 AM" src="https://github.com/user-attachments/assets/460a24ba-279b-4e48-ae0d-16a4911c5e8c" />

Reply:
Great use of screenshot!

Reply:
So at least for this first class I believe we already have a CSS class to apply this background color. And if we don't we should probably make the class in backgrounds.scss

Reply:
Good point. I was a little hesitant to use it because almost everything in those [utility classes](https://github.com/newjersey/navigator.business.nj.gov/blob/main/web/src/styles/base/background-colors.scss) uses `!important`, but maybe that's just a personal bias to ignore for now.

I was curious how this is handled in other CSS utilities/libraries, and it looks Tailwind offers this sort of [middle ground modifier to conditionally add `!important`](https://v2.tailwindcss.com/docs/just-in-time-mode#built-in-important-modifier), but that's a larger discussion probably better suited for a potential refactor.

Reply:
actually, I think the override is still needed it, since the `.fit-screen-content` `div` is in `PageSkeleton`, and we don't want to mess with that background for everything. This seems like a great use case for the new-ish [`:has()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:has) pseudo-class, though!